packages/components/src/navigator/navigator-back-button/hook.ts

```diff
@@ -17,18 +17,23 @@ export function useNavigatorBackButton(
 	const {
 		onClick,
 		as = Button,
+		goToParent: goToParentProp = false,
```

Comment:
I've used a new prop in the "back" button to indicate whether it should behave like a "goBack" or "goToParent" by default. We could in theory build a another component but it felt too much.

Reply:
I don't care too much, I applied your suggestion.